src/collapse/collapse.ts

```diff
@@ -54,14 +54,11 @@ export class NgbCollapse implements OnInit, OnChanges {
     this.animation = config.animation;
   }
 
-  ngOnInit() {
-    this._element.nativeElement.classList.add('collapse');
-    this._runTransition(this.collapsed, false, false);
-  }
+  ngOnInit() { this._runTransition(this.collapsed, false).subscribe(); }
```

Comment:
I don't understand why we need the subscribe() here.

To be removed if there is no reason, or add a comment if there is one,.

Reply:
Have just removed it, thanks!